03/app.js

```diff
@@ -7,12 +7,61 @@ const c = randomNumber(min, max);
 
 console.log(a, b, c);
 
+const sum = getSum(a, b, c);
+console.log('Suma z funkcji 1: ' + sum);
 
+const isEvenFn = isEven;
+const inputIsEven = isEvenFn(sum);
+console.log('Liczba parzysta (true) / nieparzysta (false) funkcja 2: ' + inputIsEven);
 
 
+showInfo(sum, inputIsEven);
 
 
 
+function getSum(p1, p2, p3) {
+    let s1 = parseInt(p1);
+    let s2 = parseInt(p2);
+    let s3 = parseInt(p3);
+    let suma;
+
+    if (s1 > s3 && s2 > s3) {
+        suma = s1 + s2;
+    } else if (s2 > s1 && s3 > s1) {
+        suma = s2 + s3;
+    } else
+        suma = s1 + s3;
+
+    return suma;
+}
```

Comment:
Alternatywne rozwiązanie to wrzucic liczby do tablicy, posortować i zsumować dwie pierwsze z brzegu ;)